optimum/intel/openvino/modeling_base.py

```diff
@@ -737,3 +737,9 @@ def _incompatible_inputs_warning(self, inputs: Dict):
                 return f"Got unexpected inputs: `{input_name}` set to {type(inputs[input_name])} while expected to be {dtype}."
 
         return None
+
+    def __del__(self):
+        if hasattr(self, "request"):
+            del self.request
+        if hasattr(self, "model"):
+            del self.model
```

Comment:
Is this required or just to be safe?

Reply:
I added for experiment to check that process exit issues will be resolved if we remove model and request in destructor. checking that attribute exists added for safety (if somebody already remove them before)